Google.PowerShell/Container/GkeCmdlet.cs

```diff
@@ -68,7 +68,10 @@ protected void WaitForClusterOperation(Operation operation, string project, stri
 
             while (operationStatus != ContainerOperationStatus.DONE)
             {
-                Thread.Sleep(200);
+                // We don't want to have a sleep interval too short since that will
+                // send many requests to the server and this may cause some API quotas
+                // to be hit.
+                Thread.Sleep(1000);
```

Comment:
Do we want to make this configurable with a parameter or an environment variable?